website/content/en/highlights/ 2021-11-12-event-trottle-transform.md

```diff
@@ -0,0 +1,47 @@
+---
+date: "2021-11-16"
+title: "Event trottle transform released"
+description: "A guide that addresses the new event throttle transform"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+## Event trottle transform released
+
+
+We've released a new transform that provides a user the ability to control the throughput of specific event streams.
+
+Large spikes in data volume can frequently overwhelm a service, which is especially common in log data. Previously, users lacked the necessary tooling to control throughput, such as setting quota for users and user groups utilizing Vector, which not only can cause spike in costs, but also negatively impact downstream services due to the increased load. 
```

Comment:
I'm a little concerned with the use of "quotas" here, since it's not really enforcing a quota, right? We also plan to add specific quota support at some point in the future (think the ability to configure, at the sink, how much data should be sent for a given window). It's more about preventing spikes. Curious what @spencergilbert thinks.

Reply:
We did use the term quota in the docs https://master.vector.dev/docs/reference/configuration/transforms/throttle/#quotas, due to the usage in the underlying library.